src/components/Checkout/SocialShareV2.vue

```diff
@@ -22,41 +24,39 @@
 			>
 				<kv-icon name="facebook-round" title="Facebook" class="social__icon" />
 				<span>Share</span>
-			</a>
-			<a
+			</button>
+			<button
```

Comment:
Here an on the Thanks share, I made these button actual button elements and they use the showSharePopUp utility so behavior is the same